### ISSUE TITLE
modifying the resnet

### ISSUE THREAD

**Issue by bat3a** [VisualComputingInstitute/TrackR-CNN#93]
i want to tune some parameters in the resnet layer and still be able to use the "converted weights" you provided as starting point, and i checked config where it says:
 _# Freeze applies to the whole model, not just our backend, but that's fine since only the backend uses batchnorm_
but that doesn't work and they give the following error on loading:
`Assign requires shapes of both tensors to match. lhs shape= [3,3,3,64] rhs shape= [7,7,3,64]`
so is there away to freeze some weight layers?

**Comment by ahnonay** [comment]
"freeze" refers here to the statistics accumulated in the batch norm layers, so it is probably not related to what you want to do
If you only want to load some of the weights, you can either
- edit the checkpoint file and remove the weights you do not want
- replace the name of the weights in the model for which you do not want to load from the checkpoint

Keep in mind that when you change some layer in the Resnet and initialize its weights randomly, the initialization of all subsequent layers will essentially be meaningless.

**Comment by ahnonay** [comment]
Closing this for now, please reopen if needed
